### ISSUE TITLE
Wordpress and UIKit filter

### ISSUE THREAD

**Issue by papazetis** [uikit/uikit#3723]
### UIkit version
3.0.2

### Browser
Chrome Version 71.0.3578.98
Microsoft Edge 42.17134.1.0

### Steps to reproduce
Use the minimal theme by underscores.me. Enqueue UIKit style and scripts into Wordpress. 
Add the reset filter markup or any other filter markup on template file or a file that you want.
https://getuikit.com/docs/filter#reset-filter
You must login on the WP backend to see the issue. Everything is working fine when you are disconnected from WP.

### What is actually happening?
The cards are going up and down when you are clicking on the filter buttons

**Comment by stylethemes** [comment]
yes uiki t just don't play well with WordPress, I'm currently developing a wp theme and I'm using uikit for the frontend and facing so much problem and so many bugs with sticky, masonry, tooltip and more

**Comment by papazetis** [comment]
I have no problems with uikit and Wordpress except that one. I have developed several custom themes with no issues. Of course I am not using all the components that uikit supports.

**Comment by papazetis** [comment]
The problem here is the same as the tooltip bug https://github.com/uikit/uikit/issues/3728
html margin-top causes the issue

**Comment by haydnjames** [comment]
I've been experiencing issues with the Filter too in WP. When filtering the animation is jumps then corrects itself.

It does appear to be being caused by the admin bar, but I cannot seem to get the fix to tooltip to work for filter. Has anyone had any luck implementing that solution?

**Comment by jbonlinea** [comment]
Hi there, 

Hummm, I'm new using uikit with wp, but I face some issues too.

Sticky works alone, adding an offset also, but any other settings I try fails, like start, end, position, etc...

That's really painfull and I wonder what are the recommended way to set complex custom sticky element in wp

Cheers
